### ISSUE TITLE
Do mm provide the accuracy on imagenet for pretrained model

### ISSUE THREAD

**Issue by ChengHan111** [open-mmlab/mmsegmentation#1280]
Hi I find that we are using the pretrain model from mmlab. However, I didn't find the accuracy of the pretrained models on imagenet1k. (Such as you are applying ResNetv1c on fcn as pretrained model, but no info on the performance is provided, making it harder to make comparison with different approaches) I looked for the documents imply that "All pytorch-style pretrained backbones on ImageNet are train by ourselves, with the same procedure in the [paper](https://arxiv.org/pdf/1812.01187.pdf)." Any chance I could know the accuracy of them? Thanks

**Comment by MengzhangLI** [comment]
Hi, currently our pretrained models can be divided for two parts.

(1) The first part is from official codebase, such as Swin Transformer, Twins and so on. The acc on imagenet can be found on their official repo.

(2) The second part is third-party pretrained model whose acc on imagenet is not clear. Sorry to tell you I don't know resnet v1c actual result. If I found results I would provide it with you.

By the way, this year we would support many backbones from [MMClassification](https://github.com/open-mmlab/mmclassification), whose config files recorded acc, for example, [results of ConvNext](https://github.com/open-mmlab/mmclassification/tree/master/configs/convnext#imagenet-1k).

**Comment by ChengHan111** [comment]
Thanks for your response. I try to reproduce the segmentation results as close as possible. So I am curious about the pretrain models from mmlab. I was thinking they were generated from mmclssification but it seems they are trained on some unknown settings now.

**Comment by MengzhangLI** [comment]
Yes, you are right. Currently those mmseg pretrained models are not from mmcls. Not been specified trained as pretrained benchmark.

In 2022, if possible we would make new high performance benchmark with detailed numerical results and use more backbone from MMClassification.
